tests/perf/logs/test_transformation_full.py

```diff
@@ -23,7 +23,7 @@
 from util.context import Context
 
 BATCH_METADATA = BatchMetadata("444000444", "us-east-1", "aws")
-CONTEXT = Context("function-name", "dt-url", "dt-token", False, False, "log.forwarder")
+CONTEXT = Context("function-name", "dt-url", "dt-token", False, False, "log.forwarder", 8192)
```

Comment:
A magic number, Imho consider changing the 8192 into a constant, eg 
``` DYNATRACE_LOG_INGEST_CONTENT_MAX_LENGTH = 8192 ``` and import it for each usage of this value.